nebula/ui/components/VPNMainImage.qml

```diff
@@ -367,47 +367,11 @@ Rectangle {
         }
     }
 
-    Image {
-        id: globe
-        source: "qrc:/nebula/resources/globe.svg";
-        sourceSize.height: logo.height
-        sourceSize.width: logo.width
-        visible: false
-    }
-
     Image {
         id: globeFallback
```

Comment:
Nit: Maybe we should use another name for `id` since it’s not the fallback graphic anymore.